metacat-common/src/test/resources/org/dataone/configuration/test.properties

```diff
@@ -1,4 +1,4 @@
-dbquery.enabledEngines=pathquery;solr
+dbquery.enabledEngines=solr
```

Comment:
see [this comment](https://github.com/NCEAS/metacat/pull/1726/files#diff-b255420e6f8839b7c622e62c3f859c25039069a7ad5dfae1bd8c07061ba7ae40R2207), further down the page